sonar-vbnet-plugin/src/main/java/org/sonar/plugins/vbnet/VbNetPlugin.java

```diff
@@ -66,6 +66,7 @@ public void define(Context context) {
       ReportPathCollector.class,
       // sensor
       DotNetSensor.class,
+      VbNetFileCacheSensor.class,
```

Comment:
No change in imports for this one? Looks weird

Reply:
> No change in imports for this one? Looks weird

They are in the same package: `org.sonar.plugins.vbnet`.